packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -493,7 +493,7 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
           },
       );
 
-      WidgetInspectorService.instance.initServiceExtensions(registerServiceExtension);
+      WidgetInspectorService.instance!.initServiceExtensions(registerServiceExtension);
```

Comment:
Why did this change from non-nullable to nullable?

Reply:
weird, probably an over-zealous search and replace at some point.